### PR TITLE
Dogfooding fixes

### PR DESCRIPTION
**Pull request by PauloSousapt**
This PR fixes #<issue-number>

## Tests

- [x] Tested on mobile
- [x] Tested on dektop

This PR should be manually tested by @hugomarquesdev, included on reviewers.

### CODE REVIEW

File: src/views/Profile/index.tsx [hunk 1]
```diff
@@ -63,10 +63,10 @@ const Profile: React.FC<{ isLoading?: boolean }> = props => {
...
     const signAndUpdate = async (data: any) => {
         try {
-            const messageToSign = `I prove my identity by signing this. Current timestamp is ${timestamp}`;
+            const messageToSign = `${config.signatureMessage} ${timestamp}`;
 
             await signMessage(messageToSign)
-                .then((signature) => {
+                .then((signature: string) => {
                     registerSignature(signature, messageToSign);
             })
```
Comment (psoaresbj) [inline src/views/Profile/index.tsx:71]: IMHO, we should move to a separate file in order to use it with other forms.

Comment (PauloSousapt) [reply src/views/Profile/index.tsx]: Created the handleSignature helper to handle the sign process and store its results on the cookies.

File: src/helpers/handleSignature.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                registerSignature(signature, messageToSign);
+            })
+
+        return {success: true};
+    } catch (error) {
+        console.log(error);
+
+        return {success: false};
+    }
+};
```
Comment (psoaresbj) [inline src/helpers/handleSignature.ts:20]: Add blank line in the eof.

File: src/views/Profile/index.tsx [hunk 3]
```diff
@@ -96,7 +78,9 @@ const Profile: React.FC<{ isLoading?: boolean }> = props => {
         }
         catch(e: any) {
             if (e?.data?.error?.name === 'EXPIRED_SIGNATURE') {
-                signAndUpdate(data);
+                const { success } = await handleSignature(signMessage);
```
Comment (psoaresbj) [inline src/views/Profile/index.tsx:81]: The sign message isn't always the same?
If yes, WDYT about moving the message to the method?

File: src/helpers/handleSignature.ts (hunk 2 above)
Comment (PauloSousapt) [reply src/helpers/handleSignature.ts]: Fixed.

File: src/views/Profile/index.tsx (hunk 3 above)
Comment (obernardovieira) [reply src/views/Profile/index.tsx:81]: keep in mind, we should at some point, have this message on prismic, so we can translate. But will likely be the same.

Comment (PauloSousapt) [reply src/views/Profile/index.tsx:81]: The signMessage is not the message, is the method from utils, that actually signs in the blockchain. It's a hook, i can't call it outside a component and that's the reason why i'm passing it, as a callback to the method.

**Review by hugomarquesdev — APPROVED** [review]

**Pull request merged**